bitbots_vision/config/visionparams.yaml

```diff
@@ -53,48 +37,19 @@ bitbots_vision:
-    field_color_detector_lower_values_s: 8 # Lower bound for the field color detector saturation
-    field_color_detector_lower_values_v: 20 # Lower bound for the field color detector value/brightness
-    field_color_detector_upper_values_h: 45 # Upper bound for the field color detector hue
-    field_color_detector_upper_values_s: 255 # Upper bound for the field color detector saturation
-    field_color_detector_upper_values_v: 255 # Upper bound for the field color detector value/brightness
-
-    field_boundary_detector_search_method: "downsampling_reversed" # Method for finding the field boundary (iteration, reversed, downsampling_reversed, binary)
-    field_boundary_detector_vertical_steps: 30 # Number of steps on each scanline
-    field_boundary_detector_horizontal_steps: 30 # Number of scanlines
-    field_boundary_detector_roi_height: 2 # Region Of Interest height in which we are looking for green
-    field_boundary_detector_roi_width: 2 # Region Of Interest width in which we are looking for green
-    field_boundary_detector_roi_increase: 0.1 # Value that increases the region of interest if it is located lower in the image
-    field_boundary_detector_green_threshold: 30 # Threshold of green in the area covered by the kernel
-    field_boundary_detector_head_tilt_threshold: 20 # Threshold for the dynamic search method, that describes the head angle at which we are switching between the iteration and the reversed search method.
-
-    line_detector_field_boundary_offset: 0 # Threshold in which we are also searching for lines over the field boundary
-    line_detector_use_line_points: false # Calculate and publish the line points
-    line_detector_use_line_mask: false # Calculate and publish the line mask
-    line_detector_linepoints_range: 10000 # Number of line points
-    line_detector_object_remove_grow: 1.3 # Makes objects like the ball larger before removing them from the mask
-
-    obstacle_active: false # Enables the obstacle detection
     obstacle_finder_method: "convex" # Method for the obstacle finder (distance, convex or step)
     obstacle_color_threshold: 4 # An obstacle is defined as blue/red if it contains more blue or red than this threshold (amount of pixels in per mille)
-    obstacle_white_threshold: 47 # An obstacle that contains more white than this threshold and is not colored, is an goalpost in the conventional approach (amount of pixels in per mille)
-    obstacle_field_boundary_diff_threshold: 20 # Minimal distance between detected and convex field boundary to accept it as obstacle
-    obstacle_candidate_field_boundary_offset: 30 # Fixed height of obstacles above the field boundary
     obstacle_candidate_min_width: 15 # Minimum width of an obstacle
     obstacle_candidate_max_width: 150 # Maximum width of an obstacle
     obstacle_finder_step_length: 5 # Length of an object detection step along the field boundary
```

Comment:
Are these parameters still needed?

Reply:
I can't tell, but I will try to remove them.